File: src/main/java/xyz/starsoc/object/Contests.java
package xyz.starsoc.object;

import java.io.Serializable;
import java.lang.Integer;
import java.util.List;

public class Contests implements Serializable {
  private List<Data> data;

  private Integer count;

  public List<Data> getData() {
    return this.data;
  }

  public void setData(List<Data> data) {
    this.data = data;
  }

  public Integer getCount() {
    return this.count;
  }

  public void setCount(Integer count) {
    this.count = count;
  }

  public static class Data extends ContestData implements Serializable {

  }
}

File: src/main/java/xyz/starsoc/file/Contests.kt
package xyz.starsoc.file

import net.mamoe.mirai.console.data.AutoSavePluginData
import net.mamoe.mirai.console.data.value
import xyz.starsoc.`object`.ContestData

object Contests : AutoSavePluginData("Contests"){
    var count by value(0)
    val contests : MutableMap<String, ContestData> by value(mutableMapOf())
}